pkg/gds/admin/v2/api.go

```diff
@@ -189,6 +190,27 @@ type UpdateVASPRequest struct {
 // UpdateVASPReply is identical to RetrieveVASPReply, simply renamed for clarity.
 type UpdateVASPReply RetrieveVASPReply
 
+//===========================================================================
+// Certificate Management RPCs
+//===========================================================================
+
+// Certififcate contains some high level information about a certificate issued to a
+// VASP as well as the actual trisa.gds.models.v1beta1.Certificate marshaled by
+// protojson.
+type Certificate struct {
+	SerialNumber string                 `json:"serial_number"`
+	IssuedAt     string                 `json:"issued_at"`
+	ExpiresAt    string                 `json:"expires_at"`
+	Status       string                 `json:"status"`
+	Details      map[string]interface{} `json:"details"`
+}
```

Comment:
I think this is identical to the struct we created in the BFF, so should we use this one instead of the BFF one?

Reply:
That's a good question - I guess for now I'd lean towards having an Admin API copy and a BFF copy so that one or the other could change pretty flexibly.

Reply:
Also, it makes it more obvious where to reference the struct.